doc/DeveloperGuide.md

```diff
@@ -88,6 +88,38 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+
+#### Use case: Rename Tag
+
+**MSS**
+
+1. User requests to list tags
+2. AddressBook shows a list of existing tags
+3. User requests to rename a specific tag in the list
+4. AddressBook confirms the change with the User
+5. AddressBook renames tag and updates tag for all affected persons <br>
```

Comment:
I think you're missing one step before step 5, where the `User confirms the changes.`